### PR TITLE
chore: Remove limit update step in guided tour

### LINKED ISSUE #11893: [Task]: Welcome tour updates and bugs
### Is there an existing issue for this?

- [X] I have searched the existing issues

### SubTasks

- [x] Invert the positions of the CTA's in the intro modal for new users
- [x] Remove step 1 where the user has to update the query as there is a drop off in analytics compared to other steps. Instead we prefill the query as required.
- [x] Show the response tab in the query pane always. Currently it is shown once the query run is successfull.
- [x] Fix the position of the indicator in the explorer in step 2 while indicating to click on CustomersTable widget.

### PR DESCRIPTION
**Pull request by akash-codemonk**
## Description

Invert the the buttons show in the intro modal. 
We remove the initial step where we expected the user to update the limit in a sql query as this step had a significant user drop off. 
Also the response pane will be always shown now in onboarding so that if there is an error it would be visible to the user.
Fixes the indicator position in explorer. Same as in https://github.com/appsmithorg/appsmith/pull/11472

Fixes #11893 

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

Cypress tests exist. Removed some checks accordingly.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:green_circle: Total coverage has increased</summary>

    // Code coverage diff between base branch:release and head branch: fix/remove-limit-step 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :green_circle: | total | 55.77 **(0.01)** | 37.18 **(0.02)** | 35.35 **(0)** | 56.03 **(0.01)**
 :green_circle: | app/client/src/pages/Editor/GuidedTour/useComputeCurrentStep.ts | 7.09 **(0.27)** | 0 **(0)** | 0 **(0)** | 7.09 **(0.27)**
 :red_circle: | app/client/src/pages/Editor/QueryEditor/EditorJSONtoForm.tsx | 35.81 **(-0.6)** | 28.09 **(0)** | 0 **(0)** | 37.2 **(-0.6)**
 :green_circle: | app/client/src/sagas/OnboardingSagas.ts | 21.62 **(0.57)** | 1.28 **(0.22)** | 4.35 **(0)** | 24.62 **(0.74)**
 :red_circle: | app/client/src/utils/WorkerUtil.ts | 89.76 **(0)** | 70.59 **(-1.96)** | 100 **(0)** | 93.33 **(0)**
 :green_circle: | app/client/src/utils/autocomplete/TernServer.ts | 52.94 **(0.23)** | 41.67 **(0.84)** | 36.21 **(0)** | 56.99 **(0.25)**</details>

### CODE REVIEW

File: app/client/src/pages/Editor/Explorer/Entity/index.tsx [hunk 1]
```diff
@@ -288,6 +288,7 @@ export const Entity = forwardRef(
               props.active ? "active" : ""
             } t--entity-item`}
             data-guided-tour-id={`explorer-entity-${props.name}`}
+            data-guided-tour-iid={props.name}
```
Comment (arunvjn) [inline app/client/src/pages/Editor/Explorer/Entity/index.tsx:291]: QQ; Why do we have 2 ids here ? Can we not reuse one of these ?

Comment (akash-codemonk) [reply app/client/src/pages/Editor/Explorer/Entity/index.tsx:291]: There are two different types of indicators in guided tour. There is rectangular border and square box. Each one is referring to a different id. The logic is slightly different for each of these indicators.

rectangular
square

**Review by arunvjn — APPROVED** [review]

**Pull request merged**